src/Data/NonEmpty.purs

```diff
@@ -54,7 +60,7 @@ oneOf :: forall f a. (Alternative f) => NonEmpty f a -> f a
 oneOf (NonEmpty a fa) = pure a <|> fa
 
 instance showNonEmpty :: (Show a, Show (f a)) => Show (NonEmpty f a) where
-  show (NonEmpty a fa) = "(NonEmpty " ++ show a ++ " " ++ show fa ++ ")"
+  show (NonEmpty a fa) = "NonEmpty (" ++ show a ++ ") (" ++ show fa ++ ")"
```

Comment:
We probably should have a class or member for `showsPrec`. If the goal is to print executable code, `show` really doesn't do a good job.